railties/lib/rails/commands/db/system/change/change_command.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require "rails/generators"
+require "rails/generators/rails/db/system/change/change_generator"
+
+module Rails
+  module Command
+    module Db
+      module System
+        class ChangeCommand < Base # :nodoc:
```

Comment:
For another time: seeing this deep module nesting makes me think we should consider adding something akin to Action Mailboxes' `routing` for commands.